frontend/src/scenes/web-analytics/WebAnalyticsScene.tsx

```diff
@@ -1,9 +1,16 @@
+import { PageHeader } from 'lib/components/PageHeader'
 import { SceneExport } from 'scenes/sceneTypes'
+import { WebAnalyticsLiveUserCount } from 'scenes/web-analytics/WebAnalyticsLiveUserCount'
 import { webAnalyticsLogic } from 'scenes/web-analytics/webAnalyticsLogic'
 import { WebAnalyticsDashboard } from 'scenes/web-analytics/WebDashboard'
 
 export function WebAnalyticsScene(): JSX.Element {
-    return <WebAnalyticsDashboard />
+    return (
+        <>
+            <PageHeader buttons={<WebAnalyticsLiveUserCount />} />
```

Comment:
This is pretty interesting, but IMO the nav should stay opinionated – the scene actions area is now a natural place to look for information like this, and putting this there dilutes the UX pattern of scene actions a bit. So I vote for putting it back in the scene area 😅

Reply:
Done!